test/controllers/api/v1/protected/users_controller_test.rb

```diff
@@ -39,7 +39,7 @@ class UsersControllerTest < ActionDispatch::IntegrationTest
 
           login(users(:some_administrator))
 
-          Timecop.travel((JWTSessions.access_exp_time + 3).seconds.from_now)
+          Timecop.travel((JWTSessions.access_exp_time + JWTSessions.jwt_options.leeway + 3).seconds.from_now)
```

Comment:
Because of leeway time being changed to 30 seconds (was 0) a bunch of me tests failed!!!  That's actually gratifying!!!